servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/TimeoutHttpRequesterFilter.java

```diff
@@ -60,25 +60,6 @@
     @Nullable
     private final Executor timeoutExecutor;
 
-    /**
-     * Creates a new instance which requires only that the response metadata be received before the timeout.
-     *
-     * @param duration the timeout {@link Duration}
-     */
-    public TimeoutHttpRequesterFilter(final Duration duration) {
-        this(simpleDurationTimeout(duration), false);
-    }
-
-    /**
-     * Creates a new instance which requires only that the response metadata be received before the timeout.
-     *
-     * @param duration the timeout {@link Duration}
-     * @param timeoutExecutor the {@link Executor} to use for managing the timer notifications
-     */
-    public TimeoutHttpRequesterFilter(final Duration duration, final Executor timeoutExecutor) {
```

Comment:
We can not remove these two contractors because they were present in the previous version.